src/ansys/dpf/composites/select_indices.py

```diff
@@ -56,6 +56,10 @@ def get_selected_indices(
     Notes
     -----
     Returns an empty selection if any of the collections is empty.
+
+    The indices (nodes, layers and spots) are 0-based. Pay attention to this
```

Comment:
```suggestion
    The indices (nodes, layers, and spots) are 0-based. Pay attention to this
```
Oxford comma--place a comma before the "and" or the "or" in a list of items.